pkg/config/config.go

```diff
@@ -227,6 +230,8 @@ func DefaultConfig() PluginConfig {
 }
 
 const (
-	// RuntimeUntrusted is the implicit runtime defined for ContainerdConfig.UntrustedWorkloadRuntime
+	// RuntimeUntrusted is the implicit runtime defined for "io.kubernetes.cri.untrusted-workload" annotation.
 	RuntimeUntrusted = "untrusted"
+	// RuntimeDefault is the implicit runtime defined for the default runtime.
+	RuntimeDefault = "default"
```

Comment:
I'd actually prefer not to define this. I'd rather the `runc` runtime be named `runc` to make that explicit, and just use the DefaultRuntimeName field to make that the default. My fear is that defining `default` here will make having a `default` runtime a common practice.

Reply:
I can only map `DefaultRuntime` to this, and having the default config to use runc, are you OK with that?

Reply:
Sounds good.